test/simulator/moves/substitute.js

```diff
@@ -86,4 +86,19 @@ describe('Substitute', function () {
 		battle.makeChoices('move roost', 'move dualchop');
 		assert.notStrictEqual(battle.p1.active[0].hp, battle.p1.active[0].maxhp);
 	});
+
+	it('should track what the actual damage would have been without the substitute in Gen 1', function () {
+		battle = common.gen(1).createBattle([
+			[{species: 'Ponyta', moves: ['substitute', 'growl'], evs: {hp: 252, spd: 252}}],
```

Comment:
This would probably be cleaner without EVs, but honko's calc doesn't seem to let me calculate damage for 0 evs in RBY so to come up with the `[91,108]` range I just maxed the relevant stats.